test/IRGen/autorelease.sil

```diff
@@ -5,12 +5,13 @@
 import Builtin
 import Swift
 
-// x86_64: [[OBJC:%objc_object]] = type
-// armv7:  [[OBJC:%objc_object]] = type
-// armv7s: [[OBJC:%objc_object]] = type
-// armv7k: [[OBJC:%objc_object]] = type
-// arm64:  [[OBJC:%objc_object]] = type
-// i386:   [[OBJC:%objc_object]] = type
+// x86_64:  [[OBJC:%objc_object]] = type
+// armv7:   [[OBJC:%objc_object]] = type
+// armv7s:  [[OBJC:%objc_object]] = type
+// armv7k:  [[OBJC:%objc_object]] = type
+// arm64:   [[OBJC:%objc_object]] = type
+// aarch64: [[OBJC:%objc_object]] = type
+// i386:    [[OBJC:%objc_object]] = type
```

Comment:
This seems excessive.  Why not change the `%FileCheck` call to `%FileCheck -check-prefix CHECK -check-prefix %target-cpu %s` and then use a single line with `// CHECK: [[OBJC:%objc_object] = type?

Reply:
I don’t understand the lit testing system that well. I can test what you propose. I was just trying to adapt to the existing code.